.github/workflows/coverage.yaml

```diff
@@ -11,6 +11,10 @@ on:
 jobs:
   evmone-coverage-diff:
     runs-on: ubuntu-latest
+    
+    env:
+      BLOCK_GAS_LIMIT: "36000000"
+      FILL_UNTIL: "Cancun"
```

Comment:
Just something to bear in mind.... I wonder if this will bite us at some point? I mean, if the default fill gas block limit is changed elsewhere, we might forget to change it here. Perhaps they should be defined via a config or global repo ci var.

Reply:
About config, I think we could start using the config classes to ensure that these variables are consistent a specified version of the repo (@raxhvl introduced these btw). Not easy to use in bash scripts though :sob: 
https://github.com/ethereum/execution-spec-tests/blob/5865d02c55c0835a7bcc07c6785dc799f303a734/src/config/app.py#L15-L32

Reply:
> Just something to bear in mind.... I wonder if this will bite us at some point? I mean, if the default fill gas block limit is changed elsewhere, we might forget to change it here. Perhaps they should be defined via a config or global repo ci var.

I don't think this block gas limit should go up at the same rate as the benchmarking, since the coverage in a benchmark test that uses 1M gas vs one that uses 100M should be exactly the same (It's just 99M gas of the same operations).